src/styles/components/_NotFound.scss

```diff
@@ -0,0 +1,78 @@
+$fullLength: 100%;
+
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+#notfound {
+  position: relative;
+  height: 100vh;
+  .notfound-bg {
+    position: absolute;
+    width: $fullLength;
+    height: $fullLength;
+    background-size: cover;
+  }
+  .notfound-bg:after {
+    content: '';
+    position: absolute;
+    width: $fullLength;
+    height: $fullLength;
+    background-color: #BCE0FD;
```

Comment:
This too may need to be changed. You can draw color inspiration from the existing colors in the color palette we have on the designs

Reply:
This is actually from our color palettes.

Reply:
Alright